toplevel/toploop.ml

```diff
@@ -128,6 +128,12 @@ let run_script ppf name args =
     else filename)
     | (Stdin | String _) as x -> x
   in
+  begin match !Clflags.init_file with
+  | Some f ->
+      if Sys.file_exists f then ignore (use_silently ppf (File f) )
+      else fprintf ppf "Init file not found: \"%s\".@." f
```

Comment:
should fail here, not just print

Reply:
I thought this, too - but I think the current interpretation for an interactive toplevel doesn’t either?

Reply:
No it doesn't, I copy pasted the code from the toplevel part.

Reply:
I agree completely this should be changed, but I think it should be done in another PR so that this one changes less things and is easier to merge. I will factor the code so that this line appears only once and make a PR that changes it.